internal/fuzzbinder/fuzzscraper.go

```diff
@@ -252,12 +196,7 @@ func getFuzzStatsFromScanner(lineScanner *bufio.Scanner) (*FuzzEffort, error) {
 }
 
 // getFuzzEffortFromFile gets the fuzzingEffort from a single fuzzer log file.
-func getFuzzEffortFromFile(revisionDigest intoto.DigestSet, reader io.Reader) (*FuzzEffort, error) {
-	fileBytes, err := io.ReadAll(reader)
-	if err != nil {
-		return nil, fmt.Errorf(
-			"could not read data from log file to get fuzzing effort: %v", err)
-	}
+func getFuzzEffortFromFile(revisionDigest intoto.DigestSet, fileBytes []byte) (*FuzzEffort, error) {
 	isGoodHash, err := regexp.Match(revisionDigest["sha1"], fileBytes)
```

Comment:
pre-existing: it seems that we want to do that check at least twice, e.g., in `crashDetectedInFile`. 

Does it make sense to either (i) have a function for that, such as `hasRevisionDigest`, or (ii) check before calling e.g., `getFuzzEffortFromFile`?

Reply:
I added a function for it.